src/leaseCreateCharge/saga.ts

```diff
@@ -44,7 +44,7 @@ function* fetchReceivableTypesSaga(): Generator<any, any, any> {
       switch (statusCode) {
         case 200:
           allReceivableTypes.push(...bodyAsJson.results);
-          nextUrl = bodyAsJson.next;
+          nextUrl = bodyAsJson.next || null;
```

Comment:
Is there some value in `bodyAsJson.next` that is not the next URL, or already null if there are no more receivable types?

Reply:
I just thought the value is `null` already, if there's no more data coming from backend.

Reply:
If what Juho says is correct then the or is a bit rendundant right?